src/environments/environment.prod.ts

```diff
@@ -1,3 +1,5 @@
 export const environment = {
-  production: true
+  production: true,
+  API_BASE_URL: 'http://ec2-3-92-91-194.compute-1.amazonaws.com/',
```

Comment:
If the intention is to join the base url and the port, then the ending `/` will likely need to be removed. As `http://ec2-3-92-91-194.compute-1.amazonaws.com/:8083` I believe is not valid syntax.